doc/source/train/user-guides/monitoring-logging.rst

```diff
@@ -77,6 +32,38 @@ metrics from multiple workers.
 
         Here is an example of reporting both the aggregated R2 score and mean train and validation loss from all workers.
 
-        .. literalinclude:: ../doc_code/torchmetrics_example.py
+        .. literalinclude:: ../doc_code/metric_logging.py
             :language: python
-            :start-after: __start__
+            :start-after: __torchmetrics_start__
+            :end-before: __torchmetrics_end__
+
+
+.. _train-metric-only-reporting-deprecation:
+
+(Deprecated) Reporting free-floating metrics
+--------------------------------------------
+
+Reporting metrics with ``ray.train.report(metrics, checkpoint=None)`` from every worker writes the metrics to a Ray Tune log file (``progress.csv``, ``result.json``)
+and is accessible via the ``Result.metrics_dataframe`` on the :class:`~ray.train.Result` returned by ``trainer.fit()``.
+
+As of Ray 2.43, this behavior is deprecated and will not be supported in Ray Train V2,
+which is an overhaul of Ray Train's implementation and select APIs.
```

Comment:
note to self: might be cleaner to reorganize this by v1/v2 sections to be super clear.

Reply:
oh yeah, things felt a bit confusing when I had only *certain* sections of a user guide that sh